README.md

```diff
@@ -28,7 +28,7 @@ This Github action installs PostgreSQL on the GitHub actions runner and verifies
 
 |                                         INPUT                                          |  TYPE  | REQUIRED | DEFAULT |          DESCRIPTION          |
 |----------------------------------------------------------------------------------------|--------|----------|---------|-------------------------------|
-| <a name="input_postgresql-version"></a>[postgresql-version](#input_postgresql-version) | string |   true   |         | Version of PostgreSQL. e.g 15 |
+| <a name="input_postgresql-version"></a>[postgresql-version](#input_postgresql-version) | string |   true   |         | Version of PostgreSQL. e.g 16 |
```

Comment:
You can revert this change since it gets automatically updated based on the `action.yml` description